src/solver/CMakeLists.txt

```diff
@@ -59,9 +59,6 @@ add_executable(antares-solver
 	misc/system-memory.h
 	misc/system-memory.cpp
 
-	mps/mps_define.h

```

Comment:
ok, these files were moved to **antares-deps** at some point.

Reply:
no, they were never used